integration_tests/tests/dbt_project.py

```
@@ -260,16 +260,128 @@ def seed(self, data: List[dict], table_name: str):
+            return
+
+        schema = f"default{SCHEMA_NAME_SUFFIX}" if SCHEMA_NAME_SUFFIX else "default"
+        ch_url = "http://localhost:8123"
+
+        def ch_query(query: str) -> str:
+            encoded = query.encode("utf-8")
+            req = urllib.request.Request(
+                f"{ch_url}/?user=default&password=default&mutations_sync=1",
+                data=encoded,
+            )
+            with urllib.request.urlopen(req) as resp:
```

Ruff (0.15.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)